src/__experimental__/components/form-field/form-field.component.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FormFieldStyle from './form-field.style';
+import Label from '../label';
+import FieldHelp from '../field-help';
+import OptionsHelper from '../../../utils/helpers/options-helper';
+
+const FormField = ({
+  children,
+  fieldHelp,
+  label,
+  labelAlign,
+  labelHelp,
+  labelInline,
+  labelWidth,
+  size
+}) => (
+  <FormFieldStyle>
+    { label && (
+      <Label
+        align={ labelAlign }
+        help={ labelHelp }
+        inline={ labelInline }
+        inputSize={ size }
+        labelWidth={ labelWidth }
+      >
+        { label }
+      </Label>
+    ) }
+
+    { children }
+
+    { fieldHelp && (
+      <FieldHelp labelInline={ labelInline } labelWidth={ labelWidth }>
+        { fieldHelp }
+      </FieldHelp>
+    ) }
+  </FormFieldStyle>
+);
+
+FormField.defaultProps = {
+  size: 'medium'
```

Comment:
Not sure how we could work out the default value from the options helper.

Reply:
Ok, I forgot there is an array. We can use `OptionsHelper. sizesRestricted[1]`, but it will be not readable. It will be nice to have something like enums but it is not necessary not now.